build/rst2man.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+
+# Author:
+# Contact: grubert@users.sf.net
+# Copyright: This module has been placed in the public domain.
+
+"""
+man.py
+======

```

Comment:
I noticed this too. This file is copied from [rst2man.py](https://sourceforge.net/p/docutils/code/HEAD/tree/trunk/docutils/tools/rst2man.py). It seemed reasonable to just leave the file as-is rather than make private edits to it.